lib/gui/components/file-dropper/controllers/file-dropper.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017 resin.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+module.exports = function($q, $state, SupportedFormatsModel) {
+
+  this.formats = SupportedFormatsModel;
+
+  this.isHoverState = false;
+
+  this.statePageIs = (s) => {
+    return $state.includes(s);
```

Comment:
I think we should use `$state.is()` here (https://github.com/angular-ui/ui-router/wiki/Quick-Reference#stateisstateorname--params).